integrations/timescaledb/timescaledb-metrics/README.md

```diff
@@ -0,0 +1,70 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The timescaledb-metrics integration provides a pipeline for delivering sensu metrics to a TimescaleDB table.
+
+This integration includes the following resources:
+
+* `timescaledb-metrics` [pipeline]
+* `timescaledb-metrics` [handler]
+* `timescaledb_dsn` [secret]
+* `sensu/sensu-timescaledb-handler` [asset]
+
+## Dashboards
+
+
+There are no compatible dashboards for this integration.
+
+## Setup
+
+<!-- Sensu Integration setup instructions, including Sensu agent configuration and external component configuration -->
+<!-- EXAMPLE: what configuration (if any) is required in a third-party service to enable monitoring? -->
+
+1. Set up a [TimescaleDB][timescaledb-docs] database with access for your sensu backend.
+
+    See [sensu/sensu-timescaledb-handler][sensu/sensu-timescaledb-handler-github]  docs for sensu-specific schema.
+
+2. Add the timescaledb-metrics integration and provide it with the TimescaleDB DSN.
+
+   This integration requires the following [Sensu Secrets][secrets]:
+
+   - `timescaledb_dsn`
+
+   _NOTE: this integration creates one or more Sensu Secrets using the "env" provider. The corresponding environment variables need to be set on every sensu-backend in the Sensu deployment. To add the environment variables, please modify `/etc/default/sensu-backend` or `/etc/sysconfig/sensu-backend` and restart the sensu-backend service(s)._
```

Comment:
```suggestion
```

I think we can take this out since you've configured a prompt to collect the DSN 👍